docs/api-v1-docs/projects/project-assets.md

```diff
@@ -116,6 +116,37 @@ locationType
 location
 ```
 
+## statusList
+
+Get all project assets, sorted by asset status
+```
+projects/assets/statusList.php
+```
+
+ **Parameters**
+
+Parameters are POST unless otherwise noted
+
+```
+projects_id
+```
+
+## swap
```

Comment:
Is this correct?

Reply:
the swap endpoint was documented in the file, but not copied over to the the documentation

Reply:
I think this file talks about status list twice

Reply:
Have updated! That'll teach me to copy and paste wrong!